systemtest/src/test/java/io/strimzi/systemtest/MessagingBaseST.java

```diff
@@ -168,6 +175,228 @@ int receiveMessages(int messageCount, String clusterName, boolean tlsListener, S
         return received;
     }
 
+    /**
+     * Wait for cluster availability, check availability of external routes with TLS
+     * @param userName user name
+     * @param namespace cluster namespace
+     * @param clusterName cluster name
+     * @param topicName topic name
+     * @param messageCount message count which will be send and receive by consumer and producer
+     * @throws Exception exception
+     */
+    public void waitForClusterAvailabilityTls(String userName, String namespace, String clusterName, String topicName, int messageCount) throws InterruptedException, ExecutionException, TimeoutException {
+        try (KafkaClient testClient = new KafkaClient()) {
+            Future producer = testClient.sendMessagesTls(topicName, namespace, clusterName, userName, messageCount, "SSL");
+            Future consumer = testClient.receiveMessagesTls(topicName, namespace, clusterName, userName, messageCount, "SSL");
+
+            assertThat("Producer produced all messages", producer.get(1, TimeUnit.MINUTES), is(messageCount));
+            assertThat("Consumer consumed all messages", consumer.get(1, TimeUnit.MINUTES), is(messageCount));
+        } catch (InterruptedException | ExecutionException | TimeoutException e) {
+            e.printStackTrace();
+            throw e;
+        }
+    }
+
+    /**
+     * Wait for cluster availability, check availability of external routes with SCRAM-SHA
+     * @param userName user name
+     * @param namespace cluster namespace
+     * @param clusterName cluster name
+     * @param topicName topic name
+     * @param messageCount message count which will be send and receive by consumer and producer
+     * @throws Exception exception
+     */
+    public void waitForClusterAvailabilityScramSha(String userName, String namespace, String clusterName, String topicName, int messageCount) throws InterruptedException, ExecutionException, TimeoutException {
```

Comment:
Note you're spelling it `ScramSha` (also note that this is a great example for why strict camelcase is the least-worst convention: `waitForClusterAvailabilitySCRAMSHA` and `waitForClusterAvailabilitySCRAM_SHA` are both just horrible).